Exercises/operators-and-type-conversions/operators-and-type-conversions.js

```diff
@@ -0,0 +1,11 @@
+// 1. Get `100` using `+` operator and numeric value `10`
+let num = 10;
+console.log(num + String(0))
+
+// 2. You have following string: `Amazon annual revenue of US$232.887 billion` extract revenue and count monthly income.
+let str = 'Amazon annual revenue of US$232.887 billion'
+// var regex = (/[\d|,|.|e|E|\+]+/g)
+var numEl = parseInt(str.match(/\d+/))
+console.log(numEl/12)
```

Comment:
just do round here. Good that you know the regular expression, but also you may do it by js methods rely on $ and space after numbers. You can try